src/components/attribute-picker/attribute-picker-item.js

```diff
@@ -29,39 +30,35 @@ class AttributePickerItem extends LocalizeLabsElement(LitElement) {
 				display: inline-block;
 				width: max-content;
 			}
+			:host(:focus-visible) {
+				outline: none;
+			}
 			.d2l-labs-attribute-picker-item-wrapper {
 				align-items: center;
 				background-color: var(--d2l-color-sylvite);
 				border: 1px solid var(--d2l-color-gypsum);
-				border-radius: 0.25rem;
+				border-radius: 0.3rem;
 				display: flex;
-				gap: 0.4rem;
+				gap: 0.2rem;
 				padding: 0.25rem 0.75rem 0.2rem;
 				user-select: none;
 			}
 			:host([deletable]) .d2l-labs-attribute-picker-item-wrapper {
-				padding: 0.25rem 0.6rem 0.25rem 0.55rem;
+				padding: 0.1rem 0.1rem 0.1rem 0.55rem;
+			}
+			:host([dir="rtl"][deletable]) .d2l-labs-attribute-picker-item-wrapper {
+				padding: 0.1rem 0.55rem 0.1rem 0.1rem;
```

Comment:
Optional: We've actually been slowly switching over to [CSS logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) and removing `RtlMixin`. Makes things a lot cleaner.